src/components/Templates/SlideTemplate1.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-container
+    row
+    wrap
+    fluid
+    grid-list-sm>
+    <!-- Event title card -->
+    <v-layout
+      v-show="!slideModeOn"
+      row
+      wrap
+      justify-space-between>
+      <v-flex
+        d-flex
+        xs12
+        sm8
+        md9
+        lg6>
+        <v-card
+          color="transparent"
+          flat
+          height="100px">
+          <v-card-title
+            primary
+            class="title">
+            <h2 class="display-3">{{ title }}</h2>
+          </v-card-title>
+          <v-card-title
+            primary
+            class="title">
+            <h2 class="display-2">{{ time }}</h2>
+          </v-card-title>
+        </v-card>
+      </v-flex>
+      <!-- Done Event title card -->
+
+      <!-- Carousel Component -->
```

Comment:
I don't know how complex we care to get, but so people are aware: In industry, this component would be extracted into its own file and then it would be called here to be used. 
Again, I don't think we care about this right now but it's always interesting to know what's happening :)